Lib/fontTools/varLib/varStore.py

```diff
@@ -7,6 +7,10 @@
 from collections import defaultdict
 
 
+NO_VARIATION_INDEX = 0xFFFFFFFF
```

Comment:
after we merge #2660, otTables module would contain a NO_VARIATION_INDEX constant so here you could alias to that to avoid duplication (i know it's fairly trivial)